### ISSUE TITLE
Problems with losses dropping very fast?

### ISSUE THREAD

**Issue by ychuest** [qiaoqiaoLF/MxDNA#12]
![Image](https://github.com/user-attachments/assets/18e1bcd2-cc57-4701-be96-de1f2c492bee)

I followed the pre-training approach of DNABERT, combined with the model usage style in demo.py and the tokenization design in ckpt for the pre-training replication of MxDNA. I noticed that the Loss drops very fast, is this normal?
